dotCMS/src/main/java/com/dotmarketing/image/focalpoint/FocalPointAPIImpl.java

```diff
@@ -123,7 +123,7 @@ private void writeFocalPointMeta(final Contentlet contentlet, final String field
            return parseFocalPoint(
                    (String) metadata.getCustomMeta().get(FOCAL_POINT));
        }catch (Exception e){
-          Logger.error (FocalPointAPIImpl.class, "Metadata Error retrieving focal point from custom metadata", e);
+          Logger.debug (FocalPointAPIImpl.class, "Metadata Error retrieving focal point from custom metadata", e);
```

Comment:
is debug level enough? maybe warnAndDebug?

Reply:
Thing is.. This is handled and looks horrible in our logs... It gives the wrong impression that the page is broken when it isn't. So I am trying to silence it.